Split the log message into smaller pieces when syslog is the log destination (#456)
greenbone/gvm-libs: CHANGELOG.md

Use a char pointer instead of an zero-lenght array as kb_redis struct member. [443](https://github.com/greenbone/gvm-libs/pull/443)

### Fixed
- Fixing [#434](https://github.com/greenbone/gvm-libs/pull/434) by removing the extra parentheses in `base/networking.c` [#437](https://github.com/greenbone/gvm-libs/pull/437)
- Adding initialization to struct scanner in `boreas/util_tests.c`. [#438](https://github.com/greenbone/gvm-libs/pull/438)
- Fix warnings about uninitialized variables. [#448](https://github.com/greenbone/gvm-libs/pull/448)

> - Split the message and log it in smaller pieces when syslog is the log destination.  [#455](https://github.com/greenbone/gvm-libs/pull/455)

Reviewer: ```suggestion
- Split the log message into smaller pieces when syslog is the log destination.  [#455](https://github.com/greenbone/gvm-libs/pull/455)
```

> - Split the message and log it in smaller pieces when syslog is the log destination.  [#455](https://github.com/greenbone/gvm-libs/pull/455)

Reviewer 2: ```suggestion
- Split log messages in smaller pieces when syslog is the log destination. [#455](https://github.com/greenbone/gvm-libs/pull/455)
```
